### ISSUE TITLE
Cannot navigate with <Home>, <End> key

### ISSUE THREAD

**Issue by honnip** [silverbulletmd/silverbullet#858]
I often navigate pages with the home, end keys, but I've found that doesn't work in silverbullet.
Pressing <kbd>Home</kbd> should navigate to the beginning of the page, and pressing <kbd>End</kbd> should do the opposite.

**Comment by GiantCrocodile** [comment]
I noticed a similar thing: I would like to be able to press `ESC` within a text field/block to edit the editing and get back to the rendered view (like cancel editing/undo your change). This would be similar to closing popups with `ESC` in other applications or other web apps.

**Comment by honnip** [comment]
Yes, <kbd>PageUp</kbd> and <kbd>PageDown</kbd> work as expected for me too. It scrolls a single page.

> I don't have a home or end button on my keyboard to test this, but I'd assume they'd go to the start and end of the line. This would be my expectation, they don't do that?

I too believe that if the cursor is in the editor, it should move to the start or end of the line, but if it is not, the page view should move.

**Comment by honnip** [comment]
<kbd>ESC</kbd> works fine, but I think @GiantCrocodile means the action of removing the cursor from the editor.

**Comment by GiantCrocodile** [comment]
Sorry for my late reply and that the description of what I meant wasn't clear. I would like to press `ESC` to be able to switch from this view (edit mode):
![grafik](https://github.com/silverbulletmd/silverbullet/assets/6271057/167db1ca-eb51-4b8f-a638-f79b0f9433ee)

back to (rendered view/no edit mode):

![grafik](https://github.com/silverbulletmd/silverbullet/assets/6271057/c9effdb2-9ef7-46cc-b4dc-99ed2b2e065f)

Regarding `Page down` button: When I do it at the top of the tutorial page (like where I took the screenshots from), pressing `Page Down` gets me from view/read mode into the include block: where editing is active:

```
```include
raw: "[[!silverbullet.md/Getting Started]]"
```
and when I press another time `Page Down`, I get out of this include block and to the bottom of the page. So somehow it works but it's behaving weird. Expected behavior would be to just scroll down without triggering any edit mode. `Page Up` behaves the same.
